### DISCUSSION TITLE
TorchMetrics : ModuleNotFoundError

### DISCUSSION THREAD

**Discussion by pantment** [mrdbourke/pytorch-deep-learning#424] [Q&A]
Hello, i have a problem with the torchmetrics library. I have installed in my local machine (ubuntu 22.04), i can see it with pip freeze, but when i try to import in notebook i have this error: 

ModuleNotFoundError: No module named 'torchmetrics'

I try to use another python version so i created a venv with python3.9 (i host i use python3.11) but the error is still there.

**Comment by mrdbourke** [comment]
Hi @pantelmen,

Have you tried to restart your notebook server and then run it again? 

Or even create a brand new notebook with nothing in it and try to `import torchmetrics` from there?
